indexing-service/src/test/java/io/druid/indexing/overlord/TaskLockboxTest.java

```diff
@@ -124,43 +133,32 @@ public void testTrySmallerLock() throws InterruptedException
     );
   }
 
-  @Test(expected = ISE.class)
-  public void testOverlappingIntervalForSameTask() throws InterruptedException
-  {
-    Task task = NoopTask.create();
-    lockbox.add(task);
-    Optional<TaskLock> lock1 = lockbox.tryLock(task, new Interval("2015-01-01/2015-01-03"));
-    Assert.assertTrue(lock1.isPresent());
-    Assert.assertEquals(new Interval("2015-01-01/2015-01-03"), lock1.get().getInterval());
-
-    // same task tries to take partially overlapping interval; should throw exception
-    lockbox.tryLock(task, new Interval("2015-01-02/2015-01-04"));
-  }
 
   @Test(expected = IllegalStateException.class)
   public void testTryLockForInactiveTask() throws InterruptedException
   {
     Assert.assertFalse(lockbox.tryLock(NoopTask.create(), new Interval("2015-01-01/2015-01-02")).isPresent());
   }
 
-  @Test(expected = IllegalStateException.class)
+  @Test
   public void testTryLockAfterTaskComplete() throws InterruptedException
   {
     Task task = NoopTask.create();
+    exception.expect(IllegalStateException.class);
```

Comment:
This should be ISE. Also, it would be better to check the exception message as well. Please refer to AppenderatorDriverFailTest as an example.